include/crocoddyl/core/costs/cost-sum.hpp

```diff
@@ -121,6 +121,13 @@ class CostModelSumTpl {
                boost::shared_ptr<CostModelAbstract> cost, const Scalar weight,
                const bool active = true);
 
+  /**
+   * @brief Add a cost item
+   *
+   * @param[in] cost_item Cost item
+   */
+  void addCost(const boost::shared_ptr<CostItem>& cost_item);
```

Comment:
1. We recently moved to std::shared_ptr, as you can see in this https://github.com/loco-3d/crocoddyl/pull/1339. 
2. I suggest defining `weight` (in the code above) as a default argument. This will require to update the Python bindings.
```suggestion

```

Reply:
The default value should be 1, of course.